frontend/packages/integration-tests-cypress/README.md

```diff
@@ -0,0 +1,50 @@
+#### Getting Started
+- [What is Cypress?](https://www.youtube.com/watch?v=dr10Z-HpsCQ) (video)
+- [Cypress in a Nutshell](https://www.youtube.com/watch?v=LcGHiFnBh3Y) (video)
+
+#### Best Practices
+- We are switching over our test ids from `< ... data-test-id=".."/>` to using the Cypress preferred 
+`< ... data-test=".."/>`.  This allows us to better take advantage of certain Cypress tooling, like the 
+[Selector Playground](https://docs.cypress.io/guides/core-concepts/test-runner.html#Selector-Playground) 
+- Use [Cypress's Best Practices for Selecting Elements](https://docs.cypress.io/guides/references/best-practices.html)
+
+#### Migrating Protractor tests to Cypress
+
+When migrating a test suite from Protractor to Cypress, the following steps are recommended:
+1. Create the new test suite in Cypress
+2. If you need to create a new test id use the `data-test` attribute and the `cy.getByTestID()` helper method.
+If you need to access the legacy `data-test-id` attribute, use the `cy.getByLegacyTestID()` helper method.
+3. Remove test suite from Protractor
+
+#### Directory Structure
+```
+frontend/packages/integration-tests-cypress/
```

Comment:
I have a slight preference for just `/cypress`

Does this really belong under `packages`? I don't see a `package.json`

Reply:
Your previous comment wanted them moved to `packages` so other teams would see them...have access?

Reply:
> Your previous comment wanted them moved to `packages` so other teams would see them...have access? 

We used to have a package.json file. If we put it here, we should add a package.json file like https://github.com/openshift/console/blob/master/frontend/integration-tests/package.json to make it a proper package.

We do need to think about how static plugins can contribute tests. #2051 is what we did for protractor.

Reply:
Ok, added that to the PR's todo list.